framework/instruments/src/lib.rs

```diff
@@ -53,7 +52,7 @@ impl ReportConfig {
 
     pub fn init_reporter(
         self,
-        runtime: &Runtime,
+        runtime: &tokio::runtime::Handle,
```

Comment:
I ended up having to change this to accept a Handle instead of a full Runtime, because the `wind_tunnel_instrument` macro expects an `Arc<Reporter>` field instead of an `Option<Arc<Reporter>>`. `Option::None` was used before in unit tests to circumvent this issue, but that collided with the macro.